Misc/NEWS.d/next/Library/2024-07-03-10-11-53.gh-issue-121313.D7gARW.rst

```diff
@@ -0,0 +1 @@
+Limit the reading size in the :class:`multiprocessing.Connection` class to 64KiB to prevent memory overallocation and unnecessary memory management system calls.
```

Comment:
```rst
Limit the reading size in :class:`multiprocessing.Connection <multiprocessing.connection.Connection>`
objects to 64KiB to prevent memory overallocation and unnecessary memory management system calls.
```

I don't really know where the 80-chars limit is though so feel free split at the correct position. The RST I suggested would render `multiprocessing.Connection` but with a link to `multiprocessing.connection.Connection` (which is the actual reference).